boards/arm/nucleo_f746zg/nucleo_f746zg.dts

```diff
@@ -104,8 +104,9 @@ arduino_spi: &spi1 {};
 &can1 {
 	bus-speed = <125000>;
 	sjw = <1>;
-	prop_seg_phase_seg1 = <6>;
```

Comment:
This doesn't make sense. The result here would be 17 time units instead of 11

Reply:
Thanks, my mistake. I had misread how the STM driver used that value (sum vs. same value for each). I have fixed this now by using the sum of the propagation segment and phase segment 1 set via DTS in the STM driver.

Reply:
Can you do prop-seg = <0>; and phase-seg1 = <6>; here?

Reply:
Done.